src/documentation/pages/components/picker-group/examples/_sizes-lumen-centurylink.vue

```diff
@@ -0,0 +1,138 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes-lumen-centurylink" :tabs="exampleTabs">
+    p.-text(slot="example-description") Picker groups support the following sizes: <code>-xs</code>, <code>-sm</code>, <code>-md</code>, <code>-lg</code>, <code>-xl</code>. The default size is <code>-md</code>.
+    .-d--flex.-flex--column(slot="example")
+      template(v-for="size in sizes")
+        strong -{{ size }}
+        .-py--2
+          fieldset
+            legend.chi-label Select an option
+            .chi-picker-group
+              .chi-picker-group__content
+                template(v-for="item in pickers")
+                  input.chi-picker__input(type='radio', :name="`example__size_${size}`", :id="`example__size_${size}_${item}`" :checked="item === 1")
+                  label(:for="`example__size_${size}_${item}`" :class="`-${size}`") Option {{ item }}
+          .-py--2
+            fieldset
+              legend.chi-label Select an option
+              .chi-picker-group
+                .chi-picker-group__content
+                  template(v-for="item in pickersIcons")
+                    input.chi-picker__input(type='radio', :name="`example__size_${size}_icon`", :id="`example__size_${size}_icon_${item}`" :checked="item === 4")
+                    label(:ref="`tooltip-${item}`" :for="`example__size_${size}_icon_${item}`" :class="`-${size}`" :data-tooltip="`Option ${item}`", data-position="bottom")
+                      span.-sr--only Option {{ item }}
+                      i.chi-icon.icon-atom(:class="size === 'xs' ? '-xs' : '-sm'" aria-hidden="true")
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <pre class="language-html" slot="code-htmlblueprint">
+      <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+    </pre>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
+
+@Component({
+  data: () => {
+    return {
+      pickers: [1, 2, 3],
+      pickersIcons: [4, 5, 6],
+      sizes: ['xs', 'sm', 'md', 'lg', 'xl'],
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component'
+        },
+        {
+          active: true,
+          id: 'htmlblueprint',
+          label: 'HTML blueprint'
+        }
+      ],
+      codeSnippets: {
+        webcomponent: ``,
+        htmlblueprint: ``
+      }
+    };
+  }
+})
+export default class SizesLumenCenturyLink extends Vue {
+  tooltips: any[] = [];
+
+  mounted() {
+    this.$data.pickersIcons.forEach((item: string) => {
+      this.tooltips.push(
+        chi.tooltip(this.$refs[`tooltip-${item}`] as HTMLElement)
+      );
+    });
+
+    this.setCodeSnippets();
```

Comment:
Why is this triggered after the rendering?
Can not we generate this before the component is mounted?